execution-platform.js

```diff
@@ -78,11 +84,7 @@ ExecutionPlatform.prototype.getExecutionPlatform = function() {
  * @param { ExecutionPlatformDetails } executionPlatform
  */
 ExecutionPlatform.prototype.setExecutionPlatform = function(executionPlatform) {
-  var definitions = this._bpmnjs.getDefinitions();
-  var rootElement = this._canvas.getRootElement();
-
-  this._modeling.updateModdleProperties(rootElement, definitions, {
-    [ EXECUTION_PLATFORM_NAME ]: executionPlatform.name,
-    [ EXECUTION_PLATFORM_VERSION ]: executionPlatform.version
+  this._commandStack.execute('executionPlatform.update', {
+    executionPlatform: executionPlatform
```

Comment:
Can you quickly explain why we _need_ a new command here (instead of using the existing ones)?

Reply:
There is no shared command to do this across viewers. In both decision table and the literal expression there is no command to express what we are doing here (one cannot `modeling.updateProperties(definitions)` because it causes an error). Because of that, I decided to have a custom command instead of implementing a separate one in `dmn-js`.

Reply:
Got it. Makes sense of course 👍